# DP: enable definition of wstring type, if __ENABLE_WSTRING is defined.

--- libstdc++/string~	Sat Feb 20 13:21:49 1999
+++ libstdc++/string	Sun Dec  5 16:35:27 1999
@@ -7,7 +7,9 @@
 
 extern "C++" {
 typedef basic_string <char> string;
-// typedef basic_string <wchar_t> wstring;
+#ifdef __ENABLE_WSTRING
+typedef basic_string <wchar_t> wstring;
+#endif
 } // extern "C++"
 
 #endif
